pkg/blockstorage/helpers.go

```diff
@@ -22,17 +22,19 @@ import (
 
 // Google Cloud environment variable names
 const (
-	GoogleCloudZone     = "CLOUDSDK_COMPUTE_ZONE"
-	GoogleCloudCreds    = "GOOGLE_APPLICATION_CREDENTIALS"
-	GoogleProjectID     = "projectID"
-	GoogleServiceKey    = "serviceKey"
-	AzureStorageAccount = "AZURE_STORAGE_ACCOUNT_NAME"
-	AzureStorageKey     = "AZURE_STORAGE_ACCOUNT_KEY"
-	AzureSubscriptionID = "AZURE_SUBSCRIPTION_ID"
-	AzureTenantID       = "AZURE_TENANT_ID"
-	AzureCientID        = "AZURE_CLIENT_ID"
-	AzureClentSecret    = "AZURE_CLIENT_SECRET"
-	AzureResurceGroup   = "AZURE_RESOURCE_GROUP"
+	GoogleCloudZone            = "CLOUDSDK_COMPUTE_ZONE"
+	GoogleCloudCreds           = "GOOGLE_APPLICATION_CREDENTIALS"
+	GoogleProjectID            = "projectID"
+	GoogleServiceKey           = "serviceKey"
+	AzureStorageAccount        = "AZURE_STORAGE_ACCOUNT_NAME"
+	AzureStorageKey            = "AZURE_STORAGE_ACCOUNT_KEY"
+	AzureSubscriptionID        = "AZURE_SUBSCRIPTION_ID"
+	AzureTenantID              = "AZURE_TENANT_ID"
+	AzureCientID               = "AZURE_CLIENT_ID"
+	AzureClentSecret           = "AZURE_CLIENT_SECRET"
+	AzureResurceGroup          = "AZURE_RESOURCE_GROUP"
+	AzureMigrateStorageAccount = "AZURE_MIGRATE_STORAGE_ACCOUNT_NAME"
```

Comment:
@SupriyaKasten - Why can't we just use `AzureStorageAccount` and `AzureStorageKey`

Reply:
@vkamra we use `AzureStorageAccount` and `AzureStorageKey` in our unit tests to create Azure secret from env variables. I added new ones to not be confused with these. Let me know if I should go ahead and use AzureStorageAccount & AzureStorageKey instead of the AzureMigrate variables.

Reply:
We might also need them for e2e test, to create migration profiles